daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SError.scala

```diff
@@ -48,8 +48,10 @@ object SError {
   }
 
   /** Arithmetic error such as division by zero */
-  final case class DamlEArithmeticError(message: String) extends SErrorDamlException {
-    override def toString: String = message
+  final case class DamlEArithmeticError(builtinName: String, args: ImmArray[String])
+      extends SErrorDamlException {
+    override def toString: String =
+      s"Arithmetic Exception while evaluating $builtinName(${args.iterator.mkString(",")})."
```

Comment:
```suggestion
      s"ArithmeticException while evaluating $builtinName(${args.iterator.mkString(",")})."
```
or lowercase `exception` but this seems nicer to me

Reply:
:+1:

Reply:
IMO this should be `ArithmeticError` (to match the Daml type) not `ArithmeticException`.